packages/alfa-css/src/properties/font-weight.ts

```diff
@@ -0,0 +1,61 @@
+import { parse } from "@siteimprove/alfa-lang";
+import { FontWeightGrammar } from "../grammars/font-weight";
+import { Property } from "../types";
+
+export type FontWeight = number | "bolder" | "lighter";
+
+/**
+ * @see https://www.w3.org/TR/css-fonts/#propdef-font-weight
+ */
+export const FontWeightProperty: Property<FontWeight> = {
+  inherits: true,
+  parse(input) {
+    return parse(input, FontWeightGrammar);
+  },
+  initial() {
+    return 400;
+  },
+  computed(getProperty, getParentProperty) {
+    const value = getProperty("fontWeight");
+
+    if (value !== "lighter" && value !== "bolder") {
+      return value;
+    }
+
+    const parentValue = getParentProperty("fontWeight");
+    const isBolder = value === "bolder";
+
+    switch (parentValue) {
+      case undefined:
+      case "lighter":
+      case "bolder":
+        return 400;
+    }
```

Comment:
You can get rid of the `switch` by simply doing:

```ts
if (typeof parentValue !== "number") {
  return undefined;
}
```

If `computed()` cannot determine a computed value, it should return `undefined` in which case `initial()` will kick in.

Reply:
Why not null?